radar/2019-01-01/api-management.md

```diff
@@ -5,8 +5,31 @@ quadrant:   platforms-and-partners
 
 ---
 
-## State Januar 2019 ##
+## State January 2019 ##
 
-What happened/changed in the time from 1.1.2018 till 1.1.2019
+### Evolution of API Management ###
 
-Martin Danielsson
+Since the last update of the Tech Radar, our API Management solution [wicked.haufe.io](http://wicked.haufe.io) has been applied in more projects inside the Haufe Group, and has also been adopted by other companies, such as Clarivate Analytics (part of Reuters Group).
+
+Even more now, API Management should be used as a means of decoupling systems and giving standardized access to APIs of different projects, while not having to reinvent the authentication and authorization wheel all over again. As for wicked, it is now possible to do fairly lightweight integrations with arbitrary Identity Providers, also of course including our own Foundational Services (SAML2 identity provider).
+
+wicked itself is approaching a first fully featured version 1.0; the feature set was primarily formed by the Datev integration project, [myOnboarding](https://developer.myonboarding.de), Panama (which is also transitioning to wicked from Azure API Management) and the [Developer Portal by Clarivate Analytics](https://developer.clarivate.com).

```

Comment:
If they are mentioned by name, then it's "DATEV" (all capitals); that said, the integration with DATEV's LexInform is not available to customers yet and I don't know whether we are already allowed to mention it publicly. (Need to check with Alex Bieber / Stefan Nowak).

Reply:
I'll just remove it.